csharp/src/Ice/Communicator.cs

```diff
@@ -44,11 +44,13 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
             private readonly Communicator _communicator;
         }
 
-        /// <summary>
-        /// Get the logger for this communicator.
-        /// </summary>
-        /// <returns>This communicator's logger.</returns>
-        public ILogger Logger { get; internal set; }
+         /// <summary>Indicates whether or not this communicator allows non-secure connections. When false, only
+         /// secure connections are allowed; when true, both secure and non-secure connections are allowed. This
+         /// property applies to both incoming and outgoing connections are corresponds to the Ice.AllowNonSecure
```

Comment:
` connections are corresponds to the Ice.AllowNonSecure` typo there?

Reply:
Should be "and". Fixed.